src/test/java/org/reaktivity/nukleus/tcp/internal/writer/stream/SlabTest.java

```diff
@@ -151,12 +273,12 @@ public void writtenShouldReportOutOfMemory() throws Exception
             slot = Slab.NO_SLOT;
             ByteBuffer buffer = slab.get(slot, data, 0, 5);
             int streamId = 111 + i;
-            slot = slab.written(streamId, slot, buffer);
+            slot = slab.written(streamId, slot, buffer, 0, NO_OP_WINDOW_UPDATER);
             assertTrue(slot >= 0);
         }
         slot = Slab.NO_SLOT;
         ByteBuffer buffer = slab.get(slot, data, 0, 5);
-        slot = slab.written(111, slot, buffer);
+        slot = slab.written(111, slot, buffer, 0, NO_OP_WINDOW_UPDATER);
         assertEquals(Slab.OUT_OF_MEMORY, slot);
     }
```

Comment:
Recommend inlining the `windowUpdater` mock into each test scenario for the `Slab` since those calls need to be part of the asserted `Slab.written` behavior.
Suggest removing `NO_OP_WINDOW_UPDATER` as it would be no longer needed after the change.